src/protocols-table.js

```diff
@@ -42,7 +42,8 @@ Protocols.table = [
   [477, 0, 'ws'],
   [478, 0, 'wss'],
   [275, 0, 'libp2p-webrtc-star'],
-  [276, 0, 'libp2p-webrtc-direct']
+  [276, 0, 'libp2p-webrtc-direct'],
+  [277, 0, 'p2p-circuit']
```

Comment:
@lgierth @diasdavid @whyrusleeping 

After making the change to a V in `[277, V, 'p2p-circuit']`, odd number multiaddrs break, so a valid address such as `/p2p-circuit/ipfs/QmDest` is not a valid address anymore (at least in JS). What is the reason for making it a variable length addr?